cmd/kubelet/app/options/options.go

```diff
@@ -56,11 +56,6 @@ type KubeletFlags struct {
 	KubeConfig          string
 	BootstrapKubeconfig string
 
-	// Insert a probability of random errors during calls to the master.
-	ChaosChance float64
```

Comment:
@knabben mentioned this doesn't seem to be used anywhere.
May be better to deprecate and remove after 2 releases.